top-interview-questions-medium-111-dynamic-programming-808/felix.ts

```diff
@@ -0,0 +1,32 @@
+// test cases:
+// m = 1, n = 1
+// m = 1, n = 7
+// m = 3, n = 1
+// m = 2, n = 2
+// m = 100, n = 100
+
+function uniquePaths(m: number, n: number): number {
+    // record good paths from current position, then when bot goes back to this pos, it only needs to concat with previous try
+    const map: {[m_n: string] : number } = {};
```

Comment:
what is the correct way of declaring this key using typescipt?
is it `[${m}_${n}:string]`